nixos/modules/services/mail/rspamd.nix

```diff
@@ -394,16 +394,43 @@ in
       restartTriggers = [ rspamdDir ];
 
       serviceConfig = {
-        ExecStart = "${pkgs.rspamd}/bin/rspamd ${optionalString cfg.debug "-d"} --user=${cfg.user} --group=${cfg.group} --pid=/run/rspamd.pid -c /etc/rspamd/rspamd.conf -f";
```

Comment:
I just noticed one regression, when `postfix.enable = true`, then the rspamd socket is chowned to `postfix`. However this no longer works if rspamd is started as rspamd user right away because of lacking permissions.